src/Aspire.Hosting.AWS/CloudFormation/CloudFormationAWSConsoleUrlMapper.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.AWS.CloudFormation;
+
+internal static class CloudFormationAWSConsoleUrlMapper
+{
+    public static string MapCloudFormationUrl(string stackId)
+    {
+        var region = stackId.Split(":")[3];
+
+        return
+            $"https://{region}.console.aws.amazon.com/cloudformation/home?region={region}#/stacks/resources?stackId={stackId}";
+    }
+
+    public static string? MapResourceUrl(string stackId, string resourceType, string physicalResourceId)
```

Comment:
Keeping this up to date will be an impossible task and this is one of those features that if you give it to users they are going to expect it for all of the resources. I would stick with having the URL to the stack since there isn't a way to programmatically determine the URL for any arbitrary resource. I think have the URL to the stack is a really cool addition and gives users 90 percent of what they need.

Reply:
90% is enough. Removed service specific urls.